db/migrate/20250729155036_create_lesson_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLessonTopics < ActiveRecord::Migration[8.0]
+  def change
+    create_table :lesson_topics do |t|
```

Comment:
I can't think of a reason where there would be a `lesson_topic` table entry without both a `topic_id` and a `lesson_id` so I might consider adding a `null: :false` condition here to add extra database validation. I haven't thought it all the way through though so it definitely wasn't a requirement. :)